lute/db/demo/languages/hindi.yaml

```diff
@@ -13,4 +16,4 @@ show_romanization: true
 # character_substitutions:
 split_sentences: .?!|।॥
 split_sentence_exceptions: Mr.|Mrs.|Dr.|[A-Z].|Vd.|Vds.
-word_chars: a-zA-Z\u0900-\u0963\u0966-\u097F
+word_chars: a-zA-Z\u0900-\u0963\u0966-\u097F\u200C\u200D
```

Comment:
Whoops, should have done something similar in the german.yaml file :-P  No worries, looking into it.